### ISSUE TITLE
Unable to install on the tag 3.1.16

### ISSUE THREAD

**Issue by vtsykun** [oroinc/platform#979]
Hi team,
I got an error on tag 3.1.16. I use default `composer.lock` to build own docker images for the each release (auto scan a new tag by cron) https://hub.docker.com/r/okvpn/orocommerce and noticed, that the container tests was failed on the 3.1.16 tag 

```
Fatal error: Declaration of Doctrine\ORM\Mapping\ClassMetadataFactory::wakeupReflection(Doctrine\Common\Persistence\Mapping\ClassMetadata $class, Doctrine\Common\Persistence\Mapping\ReflectionService $reflService) must be compatible with Doctrine\Persistence\Mapping\AbstractClassMetadataFactory::wakeupReflection(Doctrine\Persistence\Mapping\ClassMetadata $class, Doctrine\Persistence\Mapping\ReflectionService $reflService) in /var/www/orocommerce/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/ClassMetadataFactory.php on line 794

In ClassMetadataFactory.php line 794:
                                                                               
  Compile Error: Declaration of Doctrine\ORM\Mapping\ClassMetadataFactory::wa  
  keupReflection(Doctrine\Common\Persistence\Mapping\ClassMetadata $class, Do  
  ctrine\Common\Persistence\Mapping\ReflectionService $reflService) must be c  
  ompatible with Doctrine\Persistence\Mapping\AbstractClassMetadataFactory::w  
  akeupReflection(Doctrine\Persistence\Mapping\ClassMetadata $class, Doctrine  
  \Persistence\Mapping\ReflectionService $reflService)   
```

Thanks
Uladzimir

**Comment by mbessolov** [comment]
The problem is because of this change in doctrine/persistence - https://github.com/doctrine/persistence/commit/f8f49ec0844bef33d1ae54c36b0976666c092ea8

We will lock doctrine/persistence version in composer.json in 3.1.17

Thank you for reporting it.

_(internal ID: BAP-19548)_

**Comment by anyt** [comment]
Fixed in 3.1.17. Thank you!
